lib/private/Preview/ImaginaryPDF.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OC\Preview;
+
+class ImaginaryPDF extends Imaginary {
+	public function getMimeType(): string {
```

Comment:
```suggestion
	public static function supportedMimeTypes(): string
```

It won't work otherwise.

Reply:
9f48093b7bf23b9b3f80858528ef9b9fb62b0e7d

> It won't work otherwise.

Yes. Because of a41ed809f9c10c9532e974af95911933ded047c3 😉